recipes/default.rb

```diff
@@ -17,3 +17,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+case node['platform_family']
+when "windows"
+	include_recipe 'vcruntime::vc6'
```

Comment:
I just googled cookstyle, and im not sure i understand it or how to use it. help would be appreciated or if theres i way i can give you access to this repo on my fork and you can feel free to make changes